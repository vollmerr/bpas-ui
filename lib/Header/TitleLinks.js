var _templateObject = _taggedTemplateLiteral(['\n  color: #fff;\n  padding: ', 'px;\n  transition: all 0.25s;\n\n  &:hover,\n  &:focus {\n    text-decoration: none;\n    color: ', ';\n  }\n'], ['\n  color: #fff;\n  padding: ', 'px;\n  transition: all 0.25s;\n\n  &:hover,\n  &:focus {\n    text-decoration: none;\n    color: ', ';\n  }\n']);

function _taggedTemplateLiteral(strings, raw) { return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }

import React from 'react';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import theme from '../../util/theme';

var Links = styled(Link)(_templateObject, theme.size.xs, theme.color.accent.yellow);

/** Renders the title bars links */
function TitleLinks(props) {
  return React.createElement(
    Links,
    Object.assign({ to: '/help' }, props),
    'Contact Us'
  );
}

export default TitleLinks;