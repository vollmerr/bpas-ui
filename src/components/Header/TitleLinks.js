import React from 'react';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import theme from '../util/theme';

const Links = styled(Link)`
  color: #fff;
  padding: ${theme.size.xs}px;
  transition: all 0.25s;

  &:hover,
  &:focus {
    text-decoration: none;
    color: ${theme.color.accent.yellow};
  }
`;

/** Renders the title bars links */
function TitleLinks(props) {
  return (
      <Links to={'/help'} {...props}>Contact Us</Links>
  );
}

export default TitleLinks;
