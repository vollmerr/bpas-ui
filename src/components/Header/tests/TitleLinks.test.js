import React from 'react';
import { shallow } from 'enzyme';
import TitleLinks from '../TitleLinks';

describe('<TitleLinks />', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <TitleLinks />,
      { context: {history: {}} }
    );
    expect(tree).toMatchSnapshot();
  });

  xit('passes the correct props down', () => {});
});
