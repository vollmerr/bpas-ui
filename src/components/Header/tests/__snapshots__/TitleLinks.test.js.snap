// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleLinks /> renders correctly 1`] = `
<Styled(Link)
  to="/help"
>
  Contact Us
</Styled(Link)>
`;
